src/commons/utils/corio_utils.py

```diff
@@ -316,6 +318,47 @@ def get_master_details() -> tuple:
     return host, user, passwd
 
 
+# pylint: disable=broad-except
+def retries(asyncio=True, max_retry=S3_CFG.s3max_retry, retry_delay=S3_CFG.retry_delay):
```

Comment:
@DhobaleVishal Can we have a different file where we can identify which operation were retried and due to which error.

Reply:
> identify which operation were retried and due to which error.
code added,
> Can we have a different file 
I will adress this in next sprint.